universal-login-sdk/lib/config/SdkConfigDefault.ts

```diff
@@ -9,7 +9,7 @@ export const SdkConfigDefault: SdkConfig = {
     gasPrice: utils.bigNumberify(DEFAULT_GAS_PRICE)
   },
   observedTokens: [
-    ETHER_NATIVE_TOKEN
+    {address: ETHER_NATIVE_TOKEN.address}
```

Comment:
How is that better?

Reply:
This change follows update SdkConfig to 
```SdkConfig {
  paymentOptions: PaymentOptions;
  observedTokens: **ObservedToken[]**;
  observedCurrencies: ObservedCurrency[];
}
}
```
This done because as i know we shouldn't write in the config name and symbol of token. But get it from contract based on address
We can store observedTokens lin the config like `string[]`

Reply:
Config should be simple, e.g.:
```ts
observedTokens: ['0x31231..123123', ...]
``` 
is better than 
```ts
observedTokens: [{address: '0x31231..123123'}, ...]
```

It seems to me that this change is going other way around.